bin/getConfig.js

```diff
@@ -11,8 +12,8 @@ function getConfig() {
     return firefoxConfig;
   }
 
-  const envConfig = process.env.TARGET === "firefox-panel" ?
-                  firefoxConfig : developmentConfig;
+  const envConfig = process.env.TARGET === "application" ?
+                  applicationConfig : developmentConfig;
```

Comment:
yeah. I realized that that logic was duplicated right above actually. 

This should be tested by our CI.